packages/core/src/browser/addEventListener.ts

```diff
@@ -100,3 +98,28 @@ export function addEventListeners<E extends Event>(
     stop,
   }
 }
+
+let untrustedEventsReported: Set<string> | undefined
+
+function reportUntrustedEvent(event: Event) {
+  if (event.isTrusted || (window as any).jasmine) {
+    return
+  }
+
+  if (!untrustedEventsReported) {
+    untrustedEventsReported = new Set()
+  }
+
+  const eventType = event.type
+
+  if (untrustedEventsReported.has(eventType)) {
+    return
+  }
+
+  untrustedEventsReported.add(eventType)
+  addTelemetryDebug('Untrusted event', { event_type: eventType })
+}
+
+export function resetUntrustedEventsCount() {
+  untrustedEventsReported = undefined
+}
```

Comment:
Can we add Set as allowed in local-rules/disallow-side-effects? We could then:

```suggestion

const untrustedEventsReported = new Set<string>();

function reportUntrustedEvent(event: Event) {
  
  const eventType = event.type
  if (event.isTrusted || (window as any).jasmine || untrustedEventsReported.has(eventType) ) {
    return
  } 

  untrustedEventsReported.add(eventType)
  addTelemetryDebug('Untrusted event', { event_type: eventType })
}

export function resetUntrustedEventsCount() {
  untrustedEventsReported.clear()
}
```